redfish-core/lib/FabricAdapters.hpp

```diff
@@ -138,6 +138,32 @@ inline void
                     "LocationCode",
                     "LocationCode");
             }
+            else if (interface ==
+                     "xyz.openbmc_project.Inventory.Item.PCIeDevice")
+            {
+                // if the adapter also implements this interface, link the
+                // adapter schema to PCIeDevice schema for this adapter.
+                const std::string pcieDevice =
+                    sdbusplus::message::object_path(objPath).filename();
+
+                if (pcieDevice.empty())
+                {
+                    BMCWEB_LOG_ERROR << "Failed to find / in pcie device path";
```

Comment:
I think bmcweb got garbage from the mapper if this occurs, right?  is warn a more appropriate severity for an error condition in another application that was detected by this one.  does bmcweb have a warn severity?

Reply:
Yes, there is but since we are returning an internal error here to the user, I think error seems appropriate.